--- a/vicinae/src/ipc-command-handler.cpp
+++ b/vicinae/src/ipc-command-handler.cpp
@@ -33,8 +33,7 @@ proto::ext::daemon::Response *IpcCommandHandler::handleCommand(const proto::ext
 
 void IpcCommandHandler::handleUrl(const QUrl &url) {
   if (!std::ranges::contains(Omnicast::APP_SCHEMES, url.scheme())) {
-    qWarning() << "Unsupported url scheme" << url.scheme() << "Supported schemes are"
-               << Omnicast::APP_SCHEMES;
+    qWarning() << "Unsupported url scheme" << url.scheme() << "Supported schemes are" << Omnicast::APP_SCHEMES[0] << "and" << Omnicast::APP_SCHEMES[1];
     return;
   }
