--- a/crates/acp_thread/src/acp_thread.rs
+++ b/crates/acp_thread/src/acp_thread.rs
@@ -1947,7 +1947,7 @@
                     Ok(project.open_buffer(path, cx))
                 })
                 .map_err(|e| acp::Error::internal_error().with_data(e.to_string()))
-                .flatten()?;
+                .and_then(|x| x)?;
 
             let buffer = load.await?;
 
@@ -1956,7 +1956,7 @@
                     this.shared_buffers.get(&buffer.clone()).cloned()
                 })
                 .log_err()
-                .flatten()
+                .and_then(|x| x)
             } else {
                 None
             };
